tests/automation.js

```diff
@@ -63,6 +63,7 @@ var gfxTests = [
   { name: "gfx/CreateImmutableCopyTest", maxDifferent: 0 },
   { name: "gfx/TestLauncher", maxDifferent: 0 },
   { name: "gfx/MediaImageTest", maxDifferent: 0 },
+  { name: "gfx/TextEditorGfxTest", maxDifferent: 592 },
```

Comment:
On my Mac, the difference is 921 pixels:
![texteditorgfxtest-mac](https://cloud.githubusercontent.com/assets/305455/5601203/2f5a00f6-92a7-11e4-997d-ac4c786265e9.png)